indico/modules/events/forms.py

```diff
@@ -120,6 +120,11 @@ class EventCreationForm(EventCreationFormBase):
     end_dt = IndicoDateTimeField(_('End'), [InputRequired(), LinkedDateTime('start_dt', not_equal=True)],
                                  default_time=time(18), allow_clear=False)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.start_dt.disabled_days = [0, 5]
+        self.start_dt.disabled_dates = [date(2025, 7, 27), '2025-07-30']
```

Comment:
Hi @ThiefMaster , this is an example code, the disabled dates can be as a type `date` or string in ISO format, both work for React, but for the validation - 

```py
if `self.data` and self.disabled_dates and self.data in self.disabled_dates:
            raise StopValidation(_('Disabled date selected'))
 ```
 Do we want to enforce disabled_dates to be a date format? Or we can do
 ```py
 if self.data and self.disabled_dates and self.data.isoformat() in self.disabled_dates:
            raise StopValidation(_('Disabled date selected'))
 ```

Reply:
I would only allow `date` objects on the Python side, and no strings.